cli/commands/preview/tests/create.test.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import nock from 'nock';
+import { PreviewCreateCommand } from 'cli/commands/preview/create';
+
+jest.mock( 'fs' );
+
+describe( 'PreviewCreateCommand', () => {
+	let command: PreviewCreateCommand;
+	const mockFolder = '/test/folder';
+	const mockArchivePath = path.join( os.tmpdir(), `${ mockFolder }.zip` );
+
+	beforeEach( () => {
+		jest.clearAllMocks();
+		nock.cleanAll();
+		command = new PreviewCreateCommand( mockFolder, 'json' );
+	} );
```

Comment:
Works well, I am switch to code review, just initially wanted to propose mocking console.log on early stages, to keep terminal always clean:
```suggestion
	beforeEach( () => {
		jest.clearAllMocks();
		nock.cleanAll();
		command = new PreviewCreateCommand( mockFolder, 'json' );
		jest.spyOn( console, 'log' ).mockImplementation( () => {} );
	} );

	afterEach( () => {
		jest.spyOn( console, 'log' ).mockRestore();
	} );
```

Reply:
This is testing dummy code. The point is mostly just to ensure that Jest tests work in the `./cli` directory.

Reply:
Makes sense 👍